Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/README.md

```diff
@@ -1324,7 +1324,7 @@ Checks the reputation of the domain.
 >### Cofense Domain Reputation for domain www.sutomoresmestaj.net
 >|Threat ID|Threat Type|Verdict|Executive Summary|Campaign|Malware Family Description|Last Published|Threat Report|
 >|---|---|---|---|---|---|---|---|
->| 55110 | MALWARE | Suspicious | This report is part of our Emotet/Geodo series. Emotet is a malware family that was initially formed as a banking trojan but today often downloads additional malware payloads. We process very large Emotet campaigns containing thousands of stage one documents and we often find there are a small number of unique URLs and stage two payloads in each campaign. As such, you may notice these lists contain mostly document-specific IOCs, compared with fewer unique URLs and unique stage two payloads. | Finance or Response Themed - OfficeMacro, Emotet/Geodo | Adaptable financial crimes botnet trojan with email worm and malware delivery capabilities, also known as Emotet | 2020-08-28 13:52:25 | [https:<span>//</span>www.threathq.com/api/l/activethreatreport/55110/html](https:<span>//</span>www.threathq.com/api/l/activethreatreport/55110/html) |
+>| 55110 | MALWARE | Suspicious | This report is part of our Emotet/Geodo series. Emotet is a malware family that was initially formed as a banking trojan but today often downloads additional malware payloads. We process very large Emotet campaigns containing thousands of stage one documents and we often find there are a small number of unique URLs and stage two payloads in each campaign. As such, you may notice these lists contain mostly document-specific IOCs, compared with fewer unique URLs and unique stage two payloads. | Finance or Response Themed - OfficeMacro, Emotet/Geodo | Adaptable financial crimes botnet trojan with email worm and malware delivery capabilities, also known as Emotet | 2020-08-28 13:52:25 | [https://www.threathq.com/api/l/activethreatreport/55110/html](https://www.threathq.com/api/l/activethreatreport/55110/html) |
```

Comment:
check the site is rendered properly with this update

Reply:
Its looks good
![Screen Shot 2022-02-14 at 10 39 36](https://user-images.githubusercontent.com/78307768/153830597-8632c225-5c55-4a05-965c-965d93bf24a8.png)